api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -3738,7 +3738,7 @@
         "type": "object"
       },
       "io.k8s.api.core.v1.NodeFeatures": {
-        "description": "NodeFeatures describes the set of implemented features implemented by the CRI implementation. THE FEATURES CONTAINED IN THE NodeFeatures SHOULD DEPEND ON ONLY CRI IMPLEMENTATION, BE INDEPENDENT ON RUNTIME HANDLERS, (I.E. IT SHOULD NOT REQUIRE TO INSPECT TO ANY INFORMATION FROM OCI RUNTIME-SPEC'S FEATURES).",
+        "description": "NodeFeatures describes the set of implemented features implemented by the CRI implementation. The features contained in the NodeFeatures should depend on only cri implementation, be independent of runtime handlers, (i.e. it should not require to inspect to any information from oci runtime-spec's features).",
```

Comment:
nit: Can you move the changes in this commit to the commit where they are first introduced?

Reply:
Sure, I merged/cleaned up my commits and rebased to the latest master.

Fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66